conf/modules.config

```diff
@@ -259,7 +259,24 @@ process {
         ]
     }
 
-    withName: "SAMTOOLS_INDEX_MEM" {
+    withName: BOWTIE2_ALIGN {
+        ext.args = {[ "-N ${params.mapping_bowtie2_n}",
+        "-L ${params.mapping_bowtie2_l}",
+        "--trim5 ${params.mapping_bowtie2_trim5}",
+        "--trim3 ${params.mapping_bowtie2_trim3}",
+        "--maxins ${params.mapping_bowtie2_maxins}",
+        "--rg-id ILLUMINA-${meta.library_id}",
+        "--rg SM:${meta.sample_id}",
+        "--rg PL:illumina",
+        "--rg PU:ILLUMINA-${meta.library_id}-${meta.strandedness}",
```

Comment:
Talked with Zuzana. It will be important to get the platform and instrument right, as that can affect the underlying error model when genotyping with fancier genotypers like ATLAS, GATK etc.
I suppose we should add `Platform` and `Unit` to the input TSVs, since those would be attributes per-lane/sequencing.

Reply:
I vote we hard-code to Illumina for now, open an issue stating we wanna add an extra column to the TSV, and make a PR later to tweak that for all mappers

Reply:
Sounds good to me.